src/bika/lims/controlpanel/bika_analysisprofiles.py

```diff
@@ -122,6 +122,7 @@ def folderitem(self, obj, item, index):
 
         item["replace"]["Title"] = get_link(url, value=title)
         item["Description"] = description
+        item["ProfileKey"] = obj.ProfileKey
```

Comment:
please use the object getter instead of accessing the attribute directly.

=> `obj.getProfileKey()`

Reply:
Sorry, my bad. Corrected